lib-injection/requirements.json

```diff
@@ -29,5 +29,1281 @@
       }
     ]
   },
-  "deny": []
-}
+  "deny": [
+    {
+      "id": "gem",
+      "description": "Ignore the rubygems CLI",
+      "os": null,
+      "cmds": [
+        "**/ruby"
+      ],
+      "args": [
+        {
+          "args": [
+            "**/gem"
+          ],
+          "position": 1
+        }
+      ],
+      "envars": null
+    },
+    {
+      "id": "bundle_install",
+      "description": "Ignore bundle install",
+      "os": null,
+      "cmds": [
+        "**/ruby"
+      ],
+      "args": [
+        {
+          "args": [
+            "**/bundle",
```

Comment:
Is it possible to have args _between_ the bundle and `install` args? 🤔

Reply:
Yes it is (I wanted to chime in on that and a few other things):

```
bundle --gemfile=/path/to/Gemfile.somethingsomething install
```

Unfortunately I don't thing `args` supports this kind of thing:

```
"args": [
  "*/bundle",
  "**" // any number of args, where "*" would be any arg but just one
  "install"
]
```

And there's also this situation which would be matched above yet we might or might not want to inject but really we don't know:

```
"args": [
  "*/bundle",
  "exec"
  "somethingwedontknowabout"
  "install"
]
```

Reply:
Correct, it does not support that, it's a known limitation unfortunately

Reply:
So the closest thing I can do is:

```
"args": [
  "*/bundle",
  "install"
]
"args": [
  "*/bundle",
  "*"
  "install"
]
"args": [
  "*/bundle",
  "*"
  "*"
  "install"
]
```

and hope nobody is going to use more than N args.